postgres-connector/build.gradle

```diff
@@ -4,23 +4,19 @@
  */
 apply plugin: 'java'
 
-//repositories {
-  //  maven {
-    //    url("https://mvnrepository.com/artifact/org.postgresql/postgresql")
-      //  }
-  //}
-
  configurations {
     releaseJars
 }
 
 dependencies {
-            implementation 'org.postgresql:postgresql'
-            implementation 'org.slf4j:slf4j-api'
-            implementation 'org.odpi.egeria:data-manager-client'
-            implementation 'org.odpi.egeria:data-manager-api'
-            implementation 'org.odpi.egeria:database-integrator-api'
-            implementation 'org.odpi.egeria:open-connector-framework'
+
+            implementation 'org.postgresql:postgresql:latest.integration'
+            implementation 'org.slf4j:slf4j-api:latest.integration'
+            implementation 'org.odpi.egeria:data-manager-client:2.9-SNAPSHOT'
```

Comment:
I would recommend keeping the version numbers in the top level file as constraints as it will keep the repo consistent. This is the pattern we've adopted in egeria (albeit for maven), and is the approach taken by the last gradle setup I did here.

For example in the top level build.gradle you will see:

```
   dependencies {
         constraints
                 {
                     implementation "org.postgresql:postgresql:42.2.19"
                     implementation "org.slf4j:slf4j-api:1.7.30"
                     implementation "org.odpi.egeria:data-manager-client:${egeriaVersion}"
                     implementation "org.odpi.egeria:data-manager-api:${egeriaVersion}"
                     implementation "org.odpi.egeria:database-integrator-api:${egeriaVersion}"
                     implementation "org.odpi.egeria:open-connector-framework:${egeriaVersion}"
                 }
                 ```
                 
                 You need to ensure the required artifact is listed there with a version (like dependencyManagement on maven), and then here without it